docs/reference/connector/apis/set-connector-sync-job-error-api.asciidoc

```diff
@@ -0,0 +1,58 @@
+[[set-connector-sync-job-error-api]]
+=== Set connector sync job error API
+++++
+<titleabbrev>Set connector sync job error</titleabbrev>
+++++
+
+Sets a connector sync job error.
+
+[[set-connector-sync-job-error-api-request]]
+==== {api-request-title}
+`PUT _connector/_sync_job/<connector_sync_job_id>/_error`
+
+[[set-connector-sync-job-error-api-prereqs]]
+==== {api-prereq-title}
+
+* To sync data using connectors, it's essential to have the Elastic connectors service running.
+* The `connector_sync_job_id` parameter should reference an existing connector sync job.
+
+[[set-connector-sync-job-error-api-desc]]
+==== {api-description-title}
+
+Sets the `error` field for the specified connector sync job and sets its `status` to `error`.
+
+[[get-connector-sync-job-api-path-params]]
```

Comment:
Sorry, I made a mistake in my earlier suggestion which leads to a duplicate ID. This should fix it.
```suggestion
[[get-connector-sync-job-error-api-path-params]]
```

Reply:
No worries! Thank you for fixing :)